__tests__/app.test.js

```diff
@@ -202,6 +202,7 @@ describe("GET /api/articles/:article_id", () => {
 					created_at: expect.any(String),
 					votes: expect.any(Number),
 					article_img_url: expect.any(String),
+					comment_count: expect.any(String),
```

Comment:
Rather than amending a previous test here, could we have a new one please? You may need to change the previous one to something more flexible, if it starts to fail 🙂